src/libEth/Eth.cpp

```diff
@@ -155,7 +155,9 @@ bool ValidateEthTxn(const Transaction &tx, const Address &fromAddr,
             std::to_string(DataConversion::UnpackB(tx.GetVersion())));
   }
 
-  if (tx.GetCode().size() > MAX_EVM_CONTRACT_SIZE_BYTES) {
+  // While checking the contract size, account for Hex representation
+  // with the 'EVM' prefix.
+  if (tx.GetCode().size() > 2 * MAX_EVM_CONTRACT_SIZE_BYTES + 3) {
```

Comment:
wouldn't it be better to replace that 3 with a `constexpr` to improve readability?

Reply:
or maybe strlen("EVM") or something instead of comment?

Reply:
I started with a constexpr, but it is more code to read actually. I don't know how you can compile strlen or analog at compile time without creating another function elsewhere, but I'm bad at C++.  Seems more complicated than necessary to me (which doesn't help readability!)